### DISCUSSION TITLE
OBS Integration

### DISCUSSION THREAD

**Discussion by oparada1988** [streamdeck-linux-gui/streamdeck-linux-gui#25] [Ideas / Feature Requests]
Would be cool if OBS can be integrated, I currently use the following websocket https://github.com/muesli/obs-cli and its works great. However I think the developer has dropped support for it since the last update was a year and a half ago

**Comment by Data-007** [comment]
I am not sure which flavor you run(Linux) there are several options which I dug around. I don't know if you have come across 
'Wimpysworld' Guy interesting for you he has a OBS-Studio-Portable repo which maybe of interest. https://github.com/wimpysworld/obs-studio-portable 
Also  this maybe of interest https://github.com/ymauray/quickobs

You maybe aware of these already, anyway I am interested in this obs-cli .  

Edit: Wimpysworld is on YouTube interesting guy has several videos on streamdeck and uses streamdeck-ui

thanks 

Data-007

**Comment by oparada1988** [comment]
Maybe I was not clear, the OBC-CLI allows you to control OBS using the websocket. I use it witht he streamdeck to send the commands available in the obs-cli, i.e record, stream. switch scenes etc. I looked ovber the Portable OBS and it looks pretty cool but I dont think it has a way to add commands via the streamdeck
